test/t8_cmesh_generator/t8_cmesh_parametrized_examples/t8_cmesh_params.hxx

```diff
@@ -45,7 +45,7 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 #else
 #define T8_CMESH_MAX_NUM_OF_TREES 10
 #define T8_CMESH_MAX_NUM_OF_PRISMS 10
-#define T8_CMESH_MAX_NUM_XYZ_TREES 5
+#define T8_CMESH_MAX_NUM_XYZ_TREES 4
 #endif
 
```

Comment:
Where can I find the reduction of the testcases to 888?

Reply:
you can ./test/t8_cmesh/t8_gtest_cmesh_copy once and have a look at the number of disabled test. Make sure to have the option less-tests disabled